test/makeRule.js

```diff
@@ -456,3 +456,34 @@ const parserOptions = {
     ]
   });
 }
+{
+  const options = [
+    { schemaJsonFilepath, tagName: 'gql' },
+  ];
+
+  ruleTester.run('different tag name', rule, {
+    valid: [
+      {
+        options,
+        parserOptions,
+        code: 'const x = test.gamma`{ nonExistentQuery }`',
+      },
+      {
+        options,
+        parserOptions,
+        code: 'const x = test.gql`{ nonExistentQuery }`',
```

Comment:
This is a good test which isn't currently present in #28, checking that there aren't false-positive substring matches.